basic/components/hello_command_component/spec.yaml

```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+type: command
+name: hello_command_component
```

Comment:
>hello_command_component [](http://example.com/codeflow?start=6&length=23)

Could you please change the names according to the naming convention?  

[https://github.com/Azure/azureml-assets/wiki/Asset-naming-convention](https://github.com/Azure/azureml-assets/wiki/Asset-naming-convention)

Reply:
Removed component from the name